OpenTESArena/src/Interface/ListBox.cpp

```diff
@@ -36,8 +36,8 @@ ListBox::ListBox(int x, int y, const Color &textColor, const std::vector<std::st
 		// Remove any new lines.
 		std::string trimmedElement = String::trimLines(element);
 
-		const int x = 0;
-		const int y = 0;
+		const int x_1 = 0;
+		const int y_1 = 0;
```

Comment:
These could be `textBoxX` and `textBoxY` or something like that. `x_1` and `y_1` don't really say anything meaningful.